Mathlib/Data/Setoid/Basic.lean

```diff
@@ -443,6 +443,15 @@ def correspondence (r : Setoid α) : { s // r ≤ s } ≃o Setoid (Quotient r) w
     ⟨fun h x y hs ↦ @h ⟦x⟧ ⟦y⟧ hs, fun h x y ↦ Quotient.inductionOn₂ x y fun _ _ hs ↦ h hs⟩
 #align setoid.correspondence Setoid.correspondence
 
+/-- Given two equivalence relations with `r ≤ s`, a bijection between the sum of the quotients by
+`r` on each equivalence class by `s` and the quotient by `r`. -/
+def equivSigmaFibersOfLe {r s : Setoid α} (hle : r ≤ s) :
```

Comment:
maybe call it
```suggestion
def sigmaQuotientEquivOfLe {r s : Setoid α} (hle : r ≤ s) :
```
because the sigma is on LHS, and this makes the order of words similar to that in `sigmaFiberEquiv`, which is used in the definition. I think the capitalization of `Le` is correct here, because it's the decapitalization of `le` rather than capitalization of `LE`: I think `Le` should be used to refer to a hypothesis of the form `_ ≤ _` while `LE` should be used to refer to a typeclass argument `[LE _]`. However [ExistsMulOfLE](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/Monoid/Canonical/Defs.html#ExistsMulOfLE) and [maxOfLe](https://leanprover-community.github.io/mathlib4_docs/Init/Prelude.html#maxOfLe) don't agree with me ...